src/sagemaker/clarify.py

```diff
@@ -311,7 +312,7 @@ def __init__(
         s3_analysis_config_output_path: Optional[str] = None,
         label: Optional[str] = None,
         headers: Optional[List[str]] = None,
-        features: Optional[List[str]] = None,
+        features: Optional[str] = None,
```

Comment:
This is not a backward compatible change

Reply:
The mypy type was incorrect.  When schema validation is done, it's required to be a `str`, so it would have failed anyways: https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/clarify.py#L62

Reply:
Chasing back the changes: https://github.com/aws/sagemaker-python-sdk/commit/853e4d80240e1255aa7437b1f84399beafbad493#diff-5742d4f1aec0dedeca53accab9a77e0a63cb45e6044419975002be003357df82L68
It was accidentally being changed to List[str], which should be str.
Better re-examine all these types since it's not being added from our team.